interface-definitions/protocols-mld.xml.in

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<!-- Multicast Listener Discovery (MLD) configuration -->
+<interfaceDefinition>
+  <node name="protocols">
+    <children>
+      <node name="mld" owner="${vyos_conf_scripts_dir}/protocols_mld.py">
+        <properties>
+          <help>Multicast Listener Discovery (MLD)</help>
+        </properties>
+        <children>
+          <tagNode name="interface">
+            <properties>
+              <help>MLD interface</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces</script>
+              </completionHelp>
+            </properties>
+            <children>
+              <tagNode name="join">
+                <properties>
+                  <help>MLD join multicast group</help>
+                  <valueHelp>
+                    <format>ipv6</format>
+                    <description>Multicast group address</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ipv6-address"/>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="source">
+                    <properties>
+                      <help>Source address</help>
+                      <valueHelp>
+                        <format>ipv6</format>
+                        <description>Source address</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ipv6-address"/>
+                      </constraint>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                </children>
+              </tagNode>
+              <leafNode name="version">
+                <properties>
+                  <help>MLD version</help>
+                  <completionHelp>
+                    <list>1 2</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>1</format>
+                    <description>MLD version 1</description>
+                  </valueHelp>
+                 <valueHelp>
+                    <format>2</format>
+                    <description>MLD version 2</description>
+                  </valueHelp>
```

Comment:
Should there be  a default value? And which version is default?

Reply:
Done adding the default value.